webhooks-extension/src/api/index.js

```diff
@@ -24,7 +24,10 @@ export function getSelectedRows(selectedRows) {
 
 export function getAPIRoot() {
   const { href, hash } = window.location;
-  let newHash = hash.replace('#/extensions','v1/extensions')
+  let newHash = hash.replace('#/extensions','v1/extensions');
+  if(hash.search('create') !== -1){
```

Comment:
with this approach you'll have to update this for other potential paths in future
why not just replace the whole hash with what you're expecting?

Reply:
soo basically hard code the newHash variable?